prio_processor/spark/udf.py

```diff
@@ -62,10 +87,14 @@ def _process(share):
         )
         verifier = libprio.PrioVerifier_new(server)
         packet = libprio.PrioPacketVerify1_new()
-        # share is actually a bytearray, so convert it into bytes
-        libprio.PrioVerifier_set_data(verifier, bytes(share))
-        libprio.PrioPacketVerify1_set_data(packet, verifier)
-        return libprio.PrioPacketVerify1_write(packet)
+        try:
+            # share is actually a bytearray, so convert it into bytes
+            libprio.PrioVerifier_set_data(verifier, bytes(share))
+            libprio.PrioPacketVerify1_set_data(packet, verifier)
+            data = libprio.PrioPacketVerify1_write(packet)
```

Comment:
Maybe just return it instead of assigning it to a temporary?

```suggestion
            return libprio.PrioPacketVerify1_write(packet)
```